### PR TITLE
Connect to API Project

### PR DESCRIPTION
**Pull request by Ibnballo1**
I refactor my code to connect to API. I first fetched items from the API, then refactored my addBook to connect to the API using axios and I lastly work on the remove functionality

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,43 +1,62 @@
...
+  });
+};
+
+export const removeBook = (book) => async (dispatch) => {
+  await axios.delete(`${baseURL}${book.id}`);
+  dispatch({
+    type: REMOVE_BOOK,
+    payload: book,
+  });
+};
```
Comment (elisha2kyakpo1) [inline src/redux/books/books.js]: - [ ] Kindly refactor the removeBook function to be able to remove a book from the API, currently, nothing happens when I click the remove book button.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Ibnballo1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The PR title and description looks professional ✔️
- Descriptive commit messages 👍
- The readme file is well documented and looks professional 👍
- No linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (Ibnballo1) [reply src/redux/books/books.js]: I don't know why you have the error, but I can remove book here on my machine

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Ibnballo, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**